### PR TITLE
LPS-147872 Date picker in web content doesn't have correct styles

### PR DESCRIPTION
**Pull request by DiegoHu97**
This feedback was given to me to a bug that I opened in Clay
https://github.com/liferay/clay/issues/4715#issuecomment-1058500049

Per Patrick`s comment, I decided to add this code to frontend-theme-styled module, since the component is been used in several other modules. Let me know if I should put this code in another place. Thanks!

## Problem's describe
The date picker in Web Content does not have the correct styles:
Screenshot 2022-03-03 at 15 07 53
## What are the steps to reproduce?
1. Create a Web content
2. Open the "schedule" section in the sidebar
3. Click in the date field
## What is the expected result? 
Screenshot 2022-03-03 at 15 01 13
This is how looks like in 7.3 version (expected result), which has the following CSS:
_.popover-body, .popover-content {
      border-bottom-right-radius: calc(0.25rem - 1px);
      border-bottom-left-radius: calc(0.25rem - 1px);
      color: #6b6c7e;
      padding: 0.75rem 1rem;
}_

### CODE REVIEW

**Comment by DiegoHu97** [comment]
This feedback was given to me to a bug that I opened in Clay
https://github.com/liferay/clay/issues/4715#issuecomment-1058500049

Per Patrick`s comment, I decided to add this code to frontend-theme-styled module, since the component is been used in several other modules. Let me know if I should put this code in another place. Thanks!

## Problem's describe
The date picker in Web Content does not have the correct styles:
Screenshot 2022-03-03 at 15 07 53
## What are the steps to reproduce?
1. Create a Web content
2. Open the "schedule" section in the sidebar
3. Click in the date field
## What is the expected result? 
Screenshot 2022-03-03 at 15 01 13
This is how looks like in 7.3 version (expected result), which has the following CSS:
_.popover-body, .popover-content {
      border-bottom-right-radius: calc(0.25rem - 1px);
      border-bottom-left-radius: calc(0.25rem - 1px);
      color: #6b6c7e;
      padding: 0.75rem 1rem;
}_

**Comment by victorg1991** [comment]
ci:test:sf

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_dropdowns.scss [hunk 1]
```diff
@@ -12,4 +12,4 @@ html#{$cadmin-selector} {
 
 .dropdown-menu .dropdown-item.active {
 	pointer-events: initial;
-}
```
[the remark this answers is not in the document]
Comment (DiegoHu97) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_dropdowns.scss:15]: Hello @diegonvs, the SF makes automatically this change.

[the remark this answers is not in the document]
Comment (diegonvs) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_dropdowns.scss:15]: Ok :)

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:forward

**Pull request closed without merging**